lib/providers/unknown_provider.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:yonomi_platform_sdk/yonomi-sdk.dart';
```

Comment:
Unknown provider is not needed because its action is never going to be implemented